tests/unit/amo/components/TestAddonTitle.js

```diff
@@ -156,74 +140,76 @@ describe(__filename, () => {
       id: 102,
     };
 
-    const root = render({
+    render({
       addon: createInternalAddonWithLang({
         ...fakeAddon,
         authors: [author1, author2],
       }),
-      store,
     });
 
-    const authors = root.find('.AddonTitle-author');
-
-    // First child should be a whitespace
-    expect(authors.childAt(0).text()).toEqual(' ');
-    // Second child should be a Link
-    expect(authors.childAt(1)).toHaveProp('children', author1.name);
-    expect(authors.childAt(1)).toHaveProp('to', `/user/${author1.id}/`);
-    // Then, it should be a separator (comma)
-    expect(authors.childAt(2).text()).toEqual(' ,');
-    // Then it should be a second Link
-    expect(authors.childAt(3)).toHaveProp('children', author2.name);
-    expect(authors.childAt(3)).toHaveProp('to', `/user/${author2.id}/`);
-    // Then it should be the empty space between "by" and the links
-    expect(authors.childAt(4).text()).toEqual(' ');
-    // Finally, it should be the "by"
-    expect(authors.childAt(5).text()).toEqual('by');
+    expect(screen.getAllByRole('link')).toHaveLength(2);
+    const link1 = screen.getByRole('link', { name: author1.name });
+    expect(link1).toHaveAttribute('href', `/fa/android/user/${author1.id}/`);
+    const link2 = screen.getByRole('link', { name: author2.name });
+    expect(link2).toHaveAttribute('href', `/fa/android/user/${author2.id}/`);
+
+    expect(screen.getByClassName('AddonTitle-author')).toHaveTextContent(
+      `${author1.name} ,${author2.name} by`,
```

Comment:
Because it is RTL (real RTL this time, not `react-testing-library`). This matches the previous assertion (although the previous one is a bit hard to parse because of how it is broken up into so many pieces).

Reply:
oh, okay, I missed the test string mentioning RTL